deployment/refinery-modules/refinery/manifests/aws.pp

```diff
@@ -1,5 +1,27 @@
 class refinery::aws {
 
+# Cron job to do backups
+
+# See https://puppet.com/blog/automating-puppet-cron-jobs-and-scheduled-tasks
+# for cron.
+cron { 'cron-backup':
+  ensure => present,
+  command => 'sh /srv/refinery-platform/deployment/bin/backup-data-volume',
+  hour => ['0'],
+  minute => ['13'],     # arbitrary, but we need to pick one.
+  target => 'root',
+  user => 'root',
+}
+
+cron { 'cron-backup-rds':
+  ensure => present,
+  command => 'sh /srv/refinery-platform/deployment/bin/backup-rds',
```

Comment:
this should probably not be hard-coded (see $deployment_root: https://github.com/parklab/refinery-platform/blob/develop/deployment/manifests/aws.pp#L5)